pkg/apis/servicecatalog/v1alpha1/service_binding_storage.go

```diff
@@ -0,0 +1,15 @@
+package v1alpha1
+
+import (
```

Comment:
we put the rest storage implementations elsewhere in k8s.  I'm in favor of keeping storage and API separate from a first-principles standpoint as well.  In k8s, the rest storage impl for api group package `foo` goes into `pkg/registry/foo`, with subpackages for each resource.  So if API group `foo` has resources `Bar` and `Fiz`, you would have:

```
pkg/
  registry/
    foo/
      bar/
      fiz/
```

Refactor to that?